docs/source/markdown/podman-create.1.md

```diff
@@ -840,21 +840,28 @@ Specify the policy to select the seccomp profile. If set to *image*, Podman will
 
 Note that this feature is experimental and may change in the future.
 
-#### **\-\-secret**=*secret*
+#### **\-\-secret**=*secret*[,opt=opt ...]
 
 Give the container access to a secret. Can be specified multiple times.
 
 A secret is a blob of sensitive data which a container needs at runtime but
 should not be stored in the image or in source control, such as usernames and passwords,
 TLS certificates and keys, SSH keys or other important generic strings or binary content (up to 500 kb in size).
```

Comment:
You now talk about the secret store below, but it was never intro'd elsewhere.  IDK if this in spot on or not, but could you add something like at the end of this paragraph?

When a secret is created, it is placed in the secret store under `/run/secrets` for later reference.

Feel free to munge that further if necessary.

Reply:
Hmm, I think its best not to get too much into how the secret is stored, since the storage isn't in the container or `/run/secrets`. This is the doc for the --secret flag for a secret to be used in a container, and not for how secrets are managed. 
I think the thing that's valuable here in the docs is that secrets are written in the container at the time of container creation, and modifying the secret using podman secret commands after the container is created with the secret will not affect the secret inside the container. I've made edits, let me know if this is clearer.